ansible/install-apps.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Install OS-level apps
+  hosts: all
+  become: yes
+  vars:
+  - apt_repo_list_file: /etc/apt/sources.list.d/google-chrome.list
+  tasks:
+  - stat:
+      path: "{{ apt_repo_list_file }}"
+    register: google_apt_exists
+    ignore_errors: yes
+  - name: Add Google Chrome repo
+    copy: content={{ item.content }} dest={{ item.dest }}
```

Comment:
Use YAML mappings

Reply:
Something like that? 
```console - name: Add Google Chrome repo
    copy: content={{ item }} dest={{ apt_repo_list_file }} owner=root group=root mode=644
    with_items:
      - 'deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main'
      - 'deb http://dl.google.com/linux/chrome/deb/ stable main'
    ```

Reply:
https://yaml.org/spec/1.2/spec.html#id2798057